client/mainmenu/CMainMenu.cpp

```diff
@@ -431,22 +431,6 @@ void CMainMenu::openCampaignScreen(std::string name)
 		return;
 	}
 
-	bool campaignsFound = true;
-	for (auto const & entry : config[name]["items"].Vector())
-	{
-		ResourcePath resourceID(entry["file"].String(), EResType::CAMPAIGN);
-		if(entry["optional"].Bool())
-			continue;
-		if(!CResourceHandler::get()->existsResource(resourceID))
-			campaignsFound = false;
-	}
-
-	if (!campaignsFound)
-	{
-		CInfoWindow::showInfoDialog(LIBRARY->generaltexth->translate("vcmi.client.errors.missingCampaigns"), std::vector<std::shared_ptr<CComponent>>(), PlayerColor(1));
```

Comment:
If this string is no longer used, please remove it from translation files

Reply:
Removed obsolete string from all languages now.